source/comply/audit-log.rst

```diff
@@ -1,369 +1,293 @@
-       "disable_level", "bool", "false", "Disables output of the level name."
-       "disable_msg", "bool", "false", "Disables output of the message text."
-       "disable_fields", "bool", "false", "Disables output of all fields."
-       "disable_stacktrace", "bool", "false", "Disables output of stack traces."
-       "delim", "string", "single space", "Delimiter placed between fields."
-       "min_level_len", "number", "0", "Minimum level name length. If the level name is less than the minimum it will be padded with spaces."
-       "min_msg_len", "number", "0", "Minimum msg length. If the msg text is less than the minimum it will be padded with spaces."
-       "timestamp_format", "string", "2006-01-02 15:04:05.000 Z07:00", "Format for timestamps. See `format <https://golang.org/pkg/time/#Time.Format>`_ for format details."
-       "line_end", "string", "\n ", "Alternative end of line character(s)."
-       "enable_color", "bool", "false", "Enables color for targets that support color output."
+The following audit log levels are available:
 
-**JSON configuration options:**
++--------+-----------------------+--------------------+
+| **ID** | **Name**              | **Description**    |
++--------+-----------------------+--------------------+
+| 100    | ``audit-api``         | API events         |
++--------+-----------------------+--------------------+
+| 101    | ``audit-content``     | Content changes    |
++--------+-----------------------+--------------------+
+| 102    | ``audit-permissions`` | Permission changes |
++--------+-----------------------+--------------------+
+| 103    | ``audit-cli``         | CLI operations     |
++--------+-----------------------+--------------------+
 
-.. csv-table::
-    :header: "Key", "Type", "Default", "Description"
- 
-       "disable_timestamp", "bool", "false", "Disables output of the timestamp."
-       "disable_level", "bool", "false", "Disables output of the level name."
-       "disable_msg", "bool", "false", "Disables output of the message text."
-       "disable_fields", "bool", "false", "Disables output of all fields."
-       "disable_stacktrace", "bool", "false", "Disables output of stack traces."
-       "timestamp_format", "string", "2006-01-02 15:04:05.000 Z07:00", "Format for timestamps. See `format <https://golang.org/pkg/time/#Time.Format>`_ for format details."
+Multiple file and target support
+--------------------------------
 
- 
-**GELF configuration options:**
+System Admins can define multiple log targets to:
 
-.. csv-table::
-    :header: "Key", "Type", "Default", "Description"
-  
-       "hostname", "string", "string", "Provides a custom hostname to be output in log records, otherwise hostname is taken from the operating system."
+- Mirror log output to files and log aggregators for redundancy.
+- Log certain entries to specific destinations. For example, all errors could be routed to a specific destination for review.
```

Comment:
Sending errors to a specific destination applies to general logging, but for audits only the four levels above are used (i.e. no `error` level).  This can be re-worded to something like: 

```suggestion
- Log certain entries to specific destinations. For example, all `audit-content` records could be routed to a different destination than the other levels.
```